### PR TITLE
Add tables github_repository_dependabot_alert and github_organization_dependabot_alert

### PR DESCRIPTION
**Pull request by francois2metz**
# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

**Comment by misraved** [comment]
Great to hear that @francois2metz 👍. Let me take a take a stab at this and see if everything checks out 👍.

File: github/table_github_organization_dependabot_alert.go [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+		{
+			Name:        "dependency_package_name",
+			Type:        proto.ColumnType_STRING,
+			Description: "The unique package name within its ecosystem.",
+			Transform:   transform.FromField("Dependency.Package.Name"),
+		},
+		{
+			Name:        "dependency_manifest_path",
+			Type:        proto.ColumnType_STRING,
+			Description: "The unique package name within its ecosystem.",
```
Comment (misraved) [inline github/table_github_organization_dependabot_alert.go:42]: Could we update this description to `The unique manifestation path within the ecosystem.`?

**Review by misraved — COMMENTED** [review]
Thanks @francois2metz for adding the new tables 👍.

The structure and examples look fantastic, I have left a few minor changes, please let me know if you have questions.

**Comment by francois2metz** [comment]
I updated the PR to fix the issues.

File: github/table_github_organization_dependabot_alert.go [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func gitHubDependabotAlertColumns() []*plugin.Column {
+	return []*plugin.Column{
+		{
+			Name:        "dependabot_number",
```
Comment (cbruno10) [inline github/table_github_organization_dependabot_alert.go:17]: Is there a conflict with PostgreSQL reserved column names if we wanted to go with `number`?

File: github/table_github_organization_dependabot_alert.go [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+			Transform:   transform.FromField("Dependency.ManifestPath"),
+		},
+		{
+			Name:        "dependency_scope",
+			Type:        proto.ColumnType_STRING,
+			Description: "The execution scope of the vulnerable dependency.",
+			Transform:   transform.FromField("Dependency.Scope"),
+		},
+		{
+			Name:        "security_advisory_ghsaid",
```
Comment (cbruno10) [inline github/table_github_organization_dependabot_alert.go:52]:
```suggestion
			Name:        "security_advisory_ghsa_id",
```

Looking at the response schema in https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28#list-dependabot-alerts-for-an-enterprise, they have the field name as `ghsa_id`, so I think we should follow them.

File: github/table_github_organization_dependabot_alert.go [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+			Type:        proto.ColumnType_STRING,
+			Description: "The unique GitHub Security Advisory ID assigned to the advisory.",
+			Transform:   transform.FromField("SecurityAdvisory.GHSAID"),
+		},
+		{
+			Name:        "security_advisory_cveid",
```
Comment (cbruno10) [inline github/table_github_organization_dependabot_alert.go:58]:
```suggestion
			Name:        "security_advisory_cve_id",
```

Please see comment related to `security_advisory_ghsa_id`

File: docs/tables/github_organization_dependabot_alert.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  organization,
+  state,
+  dependency_package_ecosystem,
+  dependency_package_name
+from
+  github_organization_dependabot_alert
+where
+  organization = 'my_org'
+  and state = 'open'
+  and security_advisory_severity='critical';
```
Comment (cbruno10) [inline docs/tables/github_organization_dependabot_alert.md:51]:
```suggestion
  and security_advisory_severity = 'critical';
```

File: docs/tables/github_repository_dependabot_alert.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+select
+  state,
+  dependency_package_ecosystem,
+  dependency_package_name
+from
+  github_repository_dependabot_alert
+where
+  repository_full_name = 'turbot/steampipe'
+  and state = 'open'
+  and security_advisory_severity='critical';
```
Comment (cbruno10) [inline docs/tables/github_repository_dependabot_alert.md:48]:
```suggestion
  and security_advisory_severity = 'critical';
```

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@francois2metz Please see review comments, if you have any questions, let us know!

File: github/table_github_organization_dependabot_alert.go (hunk 2 above)
Comment (francois2metz) [reply github/table_github_organization_dependabot_alert.go:17]: I don't think so. I looked at other tables and I found that the `github_issue` table has a `issue_number`  column. I tried to follow that, but it's easy to change that.

Comment (cbruno10) [reply github/table_github_organization_dependabot_alert.go:17]: In the [API](https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28#get-a-dependabot-alert) to get a specific dependabot alert, they use the param name `alert_number`, so maybe we should follow their example and rename this column to `alert_number`. I think this new column name is a bit more descriptive, as each row in this table represents an instance of an (dependabot) alert.

**Review by misraved — APPROVED** [review]

File: github/table_github_organization_dependabot_alert.go (hunk 2 above)
Comment (misraved) [reply github/table_github_organization_dependabot_alert.go]: Thanks @francois2metz for making the updates 👍.

**Pull request merged**